sdk/identity/azure-identity/TROUBLESHOOTING.md

```diff
@@ -296,15 +296,15 @@ Get-AzAccessToken -ResourceUrl "https://management.core.windows.net"
 
 ## Troubleshoot WAM+MSA login issues
```

Comment:
Let's put the full names here with the abbreviations in parenthesis, so people searching keywords are more likely to find this document

Reply:
Here's the blurb from wikipedia that seems excellent


A Microsoft account or MSA[[1]](https://en.wikipedia.org/wiki/Microsoft_account#cite_note-1) (previously known as Microsoft Passport,[[2]](https://en.wikipedia.org/wiki/Microsoft_account#cite_note-pressp-2) .NET Passport, and Windows Live ID) is a [single sign-on](https://en.wikipedia.org/wiki/Single_sign-on) personal [user account](https://en.wikipedia.org/wiki/User_(computing)) for [Microsoft](https://en.wikipedia.org/wiki/Microsoft) customers to [log in](https://en.wikipedia.org/wiki/Login) to consumer[[3]](https://en.wikipedia.org/wiki/Microsoft_account#cite_note-3)[[4]](https://en.wikipedia.org/wiki/Microsoft_account#cite_note-4) Microsoft services (like [Outlook.com](https://en.wikipedia.org/wiki/Outlook.com)), devices running on one of Microsoft's current [operating systems](https://en.wikipedia.org/wiki/Operating_system) (e.g. [Microsoft Windows](https://en.wikipedia.org/wiki/Microsoft_Windows) computers and tablets, [Xbox](https://en.wikipedia.org/wiki/Xbox) consoles), and Microsoft [application software](https://en.wikipedia.org/wiki/Application_software) (including [Visual Studio](https://en.wikipedia.org/wiki/Visual_Studio)).

Reply:
Updated